__tests__/index.js

```diff
@@ -140,3 +150,39 @@ test('it can handle different parentKey and identifier', t => {
 
   t.deepEqual(hierarchy, expected)
 })
+
+test('check if hierarchyParser is ephemeral', t => {
+  const data = [
+    {
+      id: 1,
+      name: 'Werner Heisenberg',
+      parentId: null
+    },
+    {
+      id: 2,
+      name: 'Sommerfeld',
+      parentId: 1
+    }
+  ]
+
+  const expected = [
+    {
+      id: 1,
+      name: 'Werner Heisenberg',
+      parentId: null,
+      children: [
+        {
+          id: 2,
+          name: 'Sommerfeld',
+          parentId: 1
+        }
+      ]
+    }
+  ]
+
+  const hierarchyOne = hierarchyParser(data)
+  const hierarchyTwo = hierarchyParser(data)
+
+  t.deepEqual(hierarchyOne, expected)
+  t.deepEqual(hierarchyTwo, expected)
+})
```

Comment:
you forgot the end of line

```suggestion
})

```

Reply:
Lol